src/lib/routeLoaders.ts

```diff
@@ -105,9 +105,6 @@ export const fileLoader: LoaderFunction = async ({
     codeManager.updateCurrentFilePath(current_file_path)
     codeManager.updateCodeStateEditor(code)
 
-    // We don't want to call await on execute code since we don't want to block the UI
-    kclManager.executeCode(true)
```

Comment:
I like this change. Not `await`ing seems like asking for race conditions.

Can you remove the `kclManager` import that seems no longer used?